framework/src/araui/animation/ui_animation.c

```diff
@@ -482,7 +475,7 @@ static bool _ui_anim_rotate_func(ui_widget_t widget, ui_anim_t anim, uint32_t *d
 
 	intrp_degree = rotate_anim_body->intrp_func(anim_body->t, rotate_anim_body->from_degree, rotate_anim_body->to_degree - rotate_anim_body->from_degree, anim_body->d);
 
-	ui_widget_set_rotation(widget, intrp_degree);
+	ui_widget_set_rotation_sync((ui_widget_body_t *)widget, intrp_degree);
```

Comment:
In order to avoid svace issue, please check the return value.

Reply:
Fixed